repo/blob/s3/s3_storage_test.go

```diff
@@ -51,7 +64,36 @@ func getBucketName() string {
 	return fmt.Sprintf("kopia-test-%x", h.Sum(nil)[0:8])
 }
 
-func endpointReachable() bool {
+func generateName(name string) string {
+	b := make([]byte, 8)
```

Comment:
This can be much shorter, also the hex conversion below doubles the string size.

```suggestion
	b := make([]byte, 3)
```